packages/runtime/test/cli/compile.test.mjs

```diff
@@ -0,0 +1,52 @@
+import { test } from 'node:test'
+import assert from 'node:assert'
+import { join } from 'desm'
+import path from 'node:path'
+import { cliPath } from './helper.mjs'
+import { execa } from 'execa'
+import { mkdtemp, rm, cp, mkdir } from 'node:fs/promises'
+
+const base = join(import.meta.url, '..', 'tmp')
```

Comment:
Do we need to use this custom tmp directory instead of the tmp directory used by the other tests?

Reply:
I have removed the custom temp directory used by the other tests, they were not passing on Windows.